app/controllers/cavc_remands_controller.rb

```diff
@@ -30,24 +45,15 @@ def validate_cavc_remand_access
 
   def create_params
     params.merge!(created_by_id: current_user.id, updated_by_id: current_user.id, source_appeal_id: source_appeal.id)
-    params.require(required_params)
-    params.permit(required_params << :remand_subtype).merge(params.permit(decision_issue_ids: []))
+    params.require(required_params_by_subtype)
+    params.permit(REQUIRED_PARAMS << :remand_subtype).merge(params.permit(decision_issue_ids: []))
   end
 
-  def required_params
-    [
-      :source_appeal_id,
-      :cavc_decision_type,
-      :cavc_docket_number,
-      :cavc_judge_full_name,
-      :created_by_id,
-      :decision_date,
-      :decision_issue_ids,
-      :instructions,
-      :judgement_date,
-      :mandate_date,
-      :represented_by_attorney,
-      :updated_by_id
-    ]
+  def required_params_by_subtype
+    if params["remand_subtype"] == Constants.CAVC_REMAND_SUBTYPES.mdr
+      REQUIRED_PARAMS - [:judgement_date, :mandate_date]
```

Comment:
Convert this into a constant?

```ruby
  REQUIRED_PARAMS_FOR_MDR = (REQUIRED_PARAMS - [:judgement_date, :mandate_date]).freeze
```
```suggestion
      REQUIRED_PARAMS_FOR_MDR
```